.circleci/config.yml

```diff
@@ -46,8 +46,8 @@ jobs:
             . venv/bin/activate
             coverage run --source="." manage.py test
             coverage report --fail-under=83 -m
-            # pycodestyle ./openedx_external_enrollments
-            # pylint ./openedx_external_enrollments --rcfile=./setup.cfg
+            pycodestyle ./openedx_external_enrollments
+            pylint ./openedx_external_enrollments --rcfile=./setup.cfg
             # isort --check-only --recursive --diff ./openedx_external_enrollments
```

Comment:
What's wrong with isort? Why didn't you enable it? @andrey-canon

Reply:
isort is a different pr

Reply:
Any particular reason to include it in other PR? @andrey-canon

Reply:
too many changes

Reply:
Ok.